### PR TITLE
deprecate api operation

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

**Comment by frederic34** [comment]
image

File: htdocs/api/class/api_setup.class.php [hunk 1]
```diff
@@ -1220,7 +1220,8 @@ public function getPaymentTerms($sortfield = "sortorder", $sortorder = 'ASC', $l
 	}
 
 	/**
-	 * Get the list of shipping methods.
+	 * Get the list of shipping methods. This operation is deprecated, use
+	 * /setup/dictionary/shipment_methods instead.
```
Comment (eldy) [inline htdocs/api/class/api_setup.class.php:1224]: This url suggested seems the same than this one. ?

Comment (frederic34) [reply htdocs/api/class/api_setup.class.php:1224]: image

**Comment by eldy** [comment]
Relate to #16196

**Pull request merged**